Examples/Tests/Macroscopic_Maxwell/inputs_field_PEC_llg_exch_anis_3d

```diff
@@ -0,0 +1,126 @@
+# Set-up to test the PEC Boundary condition for the fields
+
+# max step
+max_step = 20000
+
+# number of grid points
+amr.n_cell = 32 32 32
+# amr.n_cell = 64 64 64
+
+# Maximum allowable size of each subdomain
+amr.max_grid_size = 1024
+amr.blocking_factor = 16
+
+amr.max_level = 0
+
+# Geometry
+geometry.coord_sys   = 0                     # 0: Cartesian
+geometry.prob_lo     =  -4.e-6 -4.e-6 -4.e-6
+geometry.prob_hi     =   4.e-6  4.e-6  4.e-6
+
+# Boundary condition
+boundary.field_lo = periodic periodic pec
+boundary.field_hi = periodic periodic pml
+
+warpx.serialize_ics = 1
+
+# Verbosity
+warpx.verbose = 1
+warpx.use_filter = 0
+warpx.cfl = 40000.0
```

Comment:
s the cfl large because this set-up not using Maxwell? if so, please can you leave a comment here in the input file clarifying this.

Reply:
Got it!